app/views/clients/chronic/edit.haml

```diff
@@ -5,30 +5,35 @@
 %h1= content_for :title
 = render 'clients/breadcrumbs'
 
-.mb-6
-  = render 'clients/chronic_flags', client: @client
+- if GrdaWarehouse::Config.get(:chronic_tab_roi) && !@client.active_confirmed_consent_in_cocs?(current_user.coc_codes)
+  Unable to show details due to client release status:
+  %strong= @client.release_current_status
```

Comment:
This shows nothing if they are missing ROI. I wasn't sure if we should still show the yes/no status for HUD chronicity and Potentially Chronic here @eanders ?

<img width="628" alt="Screen Shot 2022-09-19 at 5 00 11 PM" src="https://user-images.githubusercontent.com/5333982/191115617-1aead896-9fbf-4fce-b32b-502d5218d01a.png">

Reply:
This looks correct to me.  Thanks!